aioauth/config.py

```diff
@@ -0,0 +1,33 @@
+"""
+.. code-block:: python
+
+    from aioauth import config
+
+Configuration settings for aioauth server instance.
+"""
+
+from types import SimpleNamespace
+
+
+class Settings(SimpleNamespace):
```

Comment:
A quick note: For the sake of auto-documentation via Sphinx, [`types.SimpleNamespace`](https://docs.python.org/3/library/types.html#types.SimpleNamespace) is an easier object to handle. You can read up on the difference between `SimpleNamespace` and `NamedTuple` [here](https://softwareengineering.stackexchange.com/questions/349740/in-python-3-4-why-should-i-use-namedtuple-over-simplenamespace-when-not-using).

Reply:
the advantage of NamedTuple is that its instances are immutable. other advantages comparing to SimpleNamespace I don't see.

in tests to change the values of the NamedTuple attributes, I had to write a function for this that creates a new NamedTuple with new attribute values:

https://github.com/aliev/aioauth/blob/4e22bf8f1ccbae0cac0a9dbea8aeb354e3ef6e4d/tests/utils.py#L193-L195

Reply:
What are your thoughts on replacing `NamedTuple` for `SimpleNamespace`? I ask because Sphinx handles `SimpleNamespace`'s a little better.

#### `SimpleNamespace`

![Screen Shot 2021-02-24 at 12 45 44 PM](https://user-images.githubusercontent.com/2829082/109042644-71bc2c80-769e-11eb-82e2-abf2c50cbd89.png)

#### `NamedTuple`

![Screen Shot 2021-02-24 at 12 46 03 PM](https://user-images.githubusercontent.com/2829082/109042677-784aa400-769e-11eb-8568-7f72fd96aaa3.png)

Reply:
@synchronizing hm I see the difference now. SimpleNamespace looks much better :) Let's replace then :)

Reply:
Roger that. 👍🏼

Reply:
Resolved on the last commit.